site/content/docs/5.2/content/reboot.md

```diff
@@ -100,6 +100,11 @@ All heading elements—e.g., `<h1>`—and `<p>` are reset to have their `margin-
 
 The `<hr>` element has been simplified. Similar to browser defaults, `<hr>`s are styled via `border-top`, have a default `opacity: .25`, and automatically inherit their `border-color` via `color`, including when `color` is set via the parent. They can be modified with text, border, and opacity utilities.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+These `<hr>` 3rd and 4th variants can not be used because they do not respect the Orange Design System specifications. For the 3rd variant, it is because of its opacity. For the 4th variant, it is because of its height and its opacity.
+{{< /ods-incompatibility-alert >}}
+
```

Comment:
Decided:  put the callout message on top of the page above "Approach" because we don't want to use these "unstyled" examples.

New message :
This section is about how Boosted technically works. Therefore, all the examples below are not relevant regarding the Orange Design System.